engine/CMakeLists.txt

```diff
@@ -1181,17 +1190,16 @@ ELSEIF (XCODE)
     #  See https://cmake.org/cmake/help/v3.3/module/CPackDMG.html
     # Bundle -> Compressed Disk Image
     # PackageMaker - see https://cmake.org/cmake/help/v3.3/module/CPackPackageMaker.html
-    SET(CPACK_GENERATOR "DragNDrop" "BUNDLE" "TBZ2" "TGZ" "TXZ")
+    SET(CPACK_GENERATOR "DragNDrop" "BUNDLE" "TXZ")
 ELSEIF (CMAKE_SYSTEM_NAME STREQUAL Linux)
     MESSAGE("-- Configuring Packaging for Linux")
-    SET(CPACK_GENERATOR "TBZ2" "TGZ" "TXZ")
-    # Linux
+    SET(CPACK_GENERATOR "TXZ")
 
-    IF (DEFINED ENV{TAG_NAME})
-        SET(VEGA_STRIKE_PACKAGE_VERSION_STR "$ENV{TAG_NAME}")
-    ELSE (DEFINED ENV{TAG_NAME})
-        SET(VEGA_STRIKE_PACKAGE_VERSION_STR "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
-    ENDIF (DEFINED ENV{TAG_NAME})
+    # IF (DEFINED ENV{TAG_NAME})
+    #     SET(VEGA_STRIKE_PACKAGE_VERSION_STR "$ENV{TAG_NAME}")
+    # ELSE (DEFINED ENV{TAG_NAME})
+    #     SET(VEGA_STRIKE_PACKAGE_VERSION_STR "${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH}")
+    # ENDIF (DEFINED ENV{TAG_NAME})
```

Comment:
since this is above is should go away here

Reply:
Done